resolwe_api/sample.py

```diff
@@ -0,0 +1,138 @@
+"""Sample"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+import six
+
+DOWNLOAD_TYPES = {
```

Comment:
The `download()` method should also be implemented on Data and Collection. I would move this dict to `resources/base.py` and import it in `data.py`, `sample.py` and `collection.py`. I would think about a `BaseResource` class.